test/getCoverageLIb.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from "node:test";
+import assert from "node:assert";
+import { getCoverageLib } from "../src/utils";
+
+describe("getCoverageLib.js", () => {
```

Comment:
```suggestion
describe("getCoverageLib()", () => {
```
We are testing the function, not the file :p (its details but i would rename like that, it feel more accurate)